proposals/lip-0063.md

```diff
@@ -422,7 +432,10 @@ def addTokenModuleEntry():
     tokenObj.userSubstore = createUserSubstoreArray()
     tokenObj.supplySubstore = createSupplySubstoreArray()
     tokenObj.escrowSubstore = []
-    tokenObj.supportedTokensSubstore = []
+    # support all tokens 
+    tokenObj.supportedTokensSubstore = [
+        {"chainID": ALL_SUPPORTED_TOKENS_KEY, "supportedTokenIDs": []}
```

Comment:
This will currently fail when generating the genesis block to because of the schema [here](https://github.com/LiskHQ/lisk-sdk/blob/v6.0.0-beta.1/framework/src/modules/token/schemas.ts#L429-L430). As discussed with @mjerkov, it'd be better if we either use a fixed 4-byte identifier or relax the schema to support `minLength: 0`.

Reply:
Yes, we're working on changing the related schema, please see point 2. in the Description of https://github.com/LiskHQ/lips/pull/367

Reply:
@sameersubudhi  PR #367 received all necessary approvals and it relaxes the schema to allow any length for the chainId.